src/main/java/net/fabricmc/loader/impl/FabricLoaderImpl.java

```diff
@@ -77,10 +77,9 @@ public final class FabricLoaderImpl extends net.fabricmc.loader.FabricLoader {
 	public static final String VERSION = "0.15.7";
 	public static final String MOD_ID = "fabricloader";
 
-	public static final String CACHE_DIR_NAME = ".fabric"; // relative to game dir
```

Comment:
Im not super keen on replacing .fabric with this, .fabric is also used by the server installer and likely many other places that are a pain to change. I would suggest keeping fabric stuff in .fabric.

Reply:
Could maybe move the cache dir into .fabric/cache, I dont see a lot of benefit of doing otherwise?